lib/spack/spack/test/cmd/env.py

```diff
@@ -1013,6 +1009,40 @@ def test_bad_env_yaml_format_remove():
     assert badenv not in env("list")
 
 
+@pytest.mark.parametrize("answer", ["-y", ""])
+def test_multi_env_remove(mutable_mock_env_path, monkeypatch, answer):
+    """Test removal (or not) of a valid and invalid environment"""
+    resp = answer == "-y"
```

Comment:
(minor) Maybe we can use a name that reads better in the conditional below
```suggestion
    remove_environment = answer == "-y"
```